modules/cluster/variables.tf

```
@@ -29,12 +29,30 @@ variable "enabled_cluster_log_types" {
   ]
 }
 
+variable "environment" {
+  description = "Application environment for which this network is being created. Preferred value are Development, Integration, PreProduction, Production, QA, Staging, or Test"
+  type        = "string"
```

Minor change due to HCL2
```suggestion
  type        = string
```

---

Thanks, fixing